src/actions/FetchBoards.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+import { getBoardsURL } from '../api/api';
+
+ const fetchBoards = async () => {
+      try {
+        const response = await axios.get(getBoardsURL);
+        const boards = response.data;
+
+        return (boards);
```

Comment:
Does returning `(boards)` vs. `boards` behave differently?

Reply:
No, not sure how that happened. Fixed it.